content/TrueCommand/Administration/Systems.md

```diff
@@ -114,20 +121,15 @@ After adding all the desired systems to the group, click **CREATE GROUP**.
 
 Each group has two management options:
 
-+ Edit System : <i class="material-icons" aria-hidden="true" title="Configure">edit</i>
-+ Delete System : <i class="material-icons" aria-hidden="true" title="Delete">delete</i>
-
-{{< tabs "Group Control Options" >}}
-{{< tab "Edit" >}}
-Clicking the edit button <i class="material-icons" aria-hidden="true" title="Configure">edit</i> opens a side bar menu.  You can make adjustments to the Group in this manner.  Add or remove systems from the group by using the **Add System** button or the remove <i class="material-icons" aria-hidden="true" title="Delete">delete</i> button.  Click **Save Changes** when finished with your changes to update the group to the new group settings.
+{{< expand "Edit" "v" >}}
+Clicking the edit button <i class="material-icons" aria-hidden="true" title="Configure">edit</i> opens a sidebar menu. You can make adjustments to the Group in this manner. Add or remove systems from the group by using the **Add System** button or the remove <i class="material-icons" aria-hidden="true" title="Delete">delete</i> button. Click **Save Changes** when finished with your changes to update the group to the new group settings.
```

Comment:
if talking specifically about the UI screen name then capitalize it and use bold, otherwise it is lower case.
```suggestion
Clicking the edit button <i class="material-icons" aria-hidden="true" title="Configure">edit</i> opens a sidebar menu. You can make adjustments to the group in this manner. Add or remove systems from the group by using the **Add System** button or the remove <i class="material-icons" aria-hidden="true" title="Delete">delete</i> button. Click **Save Changes** when finished with your changes to update the group to the new group settings.
```